src/main/java/duke/Task.java

```diff
@@ -0,0 +1,72 @@
+package duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
```

Comment:
Great use of protected access modifier! Naming of variables are also correct so far!